saiph/reduction/famd_test.py

```diff
@@ -274,3 +275,12 @@ def test_get_variable_contributions_with_constant_variable() -> None:
     contributions, _ = get_variable_contributions(model, df, explode=False)
 
     assert np.isfinite(contributions).all().all()
+
+
+def test_reconstructed_df_from_model_equals_df(mixed_df: pd.DataFrame) -> None:
+    """Ensure that the reconstructed df from the model is equal to the original df."""
+    df = mixed_df
+    model = fit(df)
+    reconstructed_df = reconstruct_df_from_model(model)
+    # don't check dtypes, model don't know if numerical were int or float
+    assert_frame_equal(df, reconstructed_df, check_dtype=False)
```

Comment:
I think we miss some test: 
- when df is not fitted 
- when nf < nb_dim 
- what happens when row wieight/colweight are set? 
... (and other if you find others)